### DISCUSSION TITLE
Welcome to OneDark-Pro Discussions!

### DISCUSSION THREAD

**Discussion by Binaryify** [Binaryify/Material-Theme-vscode#659] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Balticbabe** [comment]
# Can anyone help me convert the hexidecimal to binary then UTF-8 or ASCII? 
`
FF E1 00 D9 45 78 69 66 00 00 4D 4D 00 2A 00 00 00 08 00 05 01 00 00 03 00 00 00 01 03 CA 00 00 01 01 00 03 00 00 00 01 08 00 00 00 87 69 00 04 00 00 00 01 00 00 00 5E 01 12 00 03 00 00 00 01 00 01 00 01 32 00 02 00 00 00 14 00 00 00 4A 00 00 00 00 32 30 32 34 3A 30 37 3A 31 34 20 30 32 3A 33 32 3A 33 37 00 00 02 92 86 00 02 00 00 00 0B 00 00 00 7C 92 08 00 03 00 00 00 01 00 00 00 00 00 00 00 52 54 5F 43 4F 4C 4C 41 47 45 00 00 04 01 00 03 00 00 00 01 03 CA 00 00 01 01 00 03 00 00 00 01 08 00 00 00 01 12 00 03 00 00 00 01 00 01 00 01 32 00 02 00 00 00 14 00 00 00 BD 00 00 00 00 32 30 32 34 3A 30 37 3A 31 34 20 30 32 3A 33 32 3A 33 37 00 FF E1 00 10 4A 46 49 46 00 01 01 00 01 00 01 00 00 FF E2 02 28 49 43 43 5F 50 52 4F 46 49 4C 45 00 01 01 00 02 18 00 00 00 00 04 30 00 00 6D 6E 74 72 52 47 42 20 58 59 5A 20 00 00 00 00 00 00 00 00 61 63 73 70 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 01 00 00 F6 D6 00 01 00 00 00 D3 2D 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
